tests/ImageSharp.Tests/Formats/Png/PngEncoderTests.cs

```diff
@@ -2,6 +2,9 @@
 // Licensed under the Apache License, Version 2.0.
 
 // ReSharper disable InconsistentNaming
```

Comment:
Can you please clean up these tests to remove the warnings. It just reduces the clutter when reading.

Reply:
sure, somehow i did not noticed that warning.

Reply:
Perfect. Thanks!